datafusion/sqllogictest/test_files/insert_to_external.slt

```diff
@@ -267,8 +267,6 @@ INSERT INTO single_file_test values (4, 5), (6, 7);
 query II
 select * from single_file_test;
 ----
-1 2
```

Comment:
This is a change worth highlighting, inserting into a ListingTable backed by a single CSV or JSON file now overwrites

Reply:
I think it should be an error. It would also be really good to call out how to get "append" semantics (which I believe requires using `CREATE UNBOUNDED TABLE ...`)?